lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
In the interest in consistency amongst our own utils, I think it should be env. `local push` is a Docker build command, just on device rather than locally. So if we changed it to `--build-arg` we'd have to change `local push` as well which is a breaking change.